components/Navigation.tsx

```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function Navigation(): JSX.Element {
+  return (
+    <Link href={"/newUserForm"}>
+      <StyledButton>create your profile</StyledButton>
+    </Link>
+  );
```

Comment:
This renders a `button` inside an `a` tag which is not valid semantical HTML. Please style your `Link` component directly to achieve the look you want to have.

Reply:
You are right, I didn´t think about it. I fixed it